data/categories.txt

```diff
@@ -177,7 +177,7 @@ ro:4Benzinărie|Benzină
 es:3Gasolinera
 sv:3Bensin
 th:ก๊าซ
-tr:Benzin
+tr:Benzinlik
```

Comment:
```suggestion
tr:3Benzinlik|Yakıt
```
You may also add some other synonyms that Turkish people may type to find a gas station. A number means "show this word as a search suggestion after 3 characters", e.g. when the user types "ben" he will see the whole word as a hint.